### DISCUSSION TITLE
NodeJS MSA Features

### DISCUSSION THREAD

**Discussion by panoti** [nashtech-garage/nodejs-msa#89] [General]
### User (Guest / merchant)
- User can log in
- User can log out
- User can sign up
- User can renew their forgot password

### Guest
- User can list rooms 
- User can view room detail
  + name
  + description
  + pics
  + host
  + address / location
  + price
  + amenities
- User can reserve the room

### Merchant
- Merchant can list their rooms
- Merchant can view their room
- Merchant can add/update/delete their room

**Comment by emhat098** [comment]
I made the wireframe for mobile. Please refer to [Wireframe mobile](https://balsamiq.cloud/sx3awc/p9rzqo1)

**Reply by emhat098** [reply]
<img width="324" alt="image" src="https://github.com/nashtech-garage/nodejs-msa/assets/108352933/c995b749-869e-41cc-bce3-93fe62cbad45">
<img width="293" alt="image" src="https://github.com/nashtech-garage/nodejs-msa/assets/108352933/830155ae-deb9-4c66-a4da-03eafbfd42e8">
<img width="298" alt="image" src="https://github.com/nashtech-garage/nodejs-msa/assets/108352933/0807352c-ccc2-4e05-88ef-f4caee898ca4">
<img width="329" alt="image" src="https://github.com/nashtech-garage/nodejs-msa/assets/108352933/ab092df1-42f6-4d87-bf31-1fea4624b690">
<img width="397" alt="image" src="https://github.com/nashtech-garage/nodejs-msa/assets/108352933/ffe79a61-6b06-4a17-b0e2-4b1e9b3c2294">
<img width="345" alt="image" src="https://github.com/nashtech-garage/nodejs-msa/assets/108352933/85c2a4c3-053e-4154-bc11-0acb312abd10">
<img width="717" alt="image" src="https://github.com/nashtech-garage/nodejs-msa/assets/108352933/e9d16467-775a-40ac-940b-f3858839b47b">
<img width="378" alt="image" src="https://github.com/nashtech-garage/nodejs-msa/assets/108352933/4fa6f5a1-a817-4d16-ac89-38699ed1116d">

**Reply by emhat098** [reply]
I just have update wireframe in mobile.

**Comment by nqbao3008** [comment]
Home front  wireframe for web
![Screenshot 2023-05-25 160055](https://github.com/nashtech-garage/nodejs-msa/assets/22763846/830576c9-5d2e-4abf-90ac-3ebe26468d71)
![image](https://github.com/nashtech-garage/nodejs-msa/assets/22763846/374b5f51-a410-456f-bb70-98f19ecd6dcb)
![image](https://github.com/nashtech-garage/nodejs-msa/assets/22763846/cd566091-0140-4252-a316-237dfb454f4c)
![image](https://github.com/nashtech-garage/nodejs-msa/assets/22763846/ad5e6cfe-7da1-447a-8656-bdce168d9e36)
![image](https://github.com/nashtech-garage/nodejs-msa/assets/22763846/063cb19e-dfbb-491f-ba8b-f1a49a29aa19)
![image](https://github.com/nashtech-garage/nodejs-msa/assets/22763846/45716231-d94b-4a38-909a-a2f8f4509408)

**Comment by nqbao3008** [comment]
merchant wireframe for web (optional)
![image](https://github.com/nashtech-garage/nodejs-msa/assets/22763846/f749de4d-07f1-4044-b31a-c1f0b280d30f)
![image](https://github.com/nashtech-garage/nodejs-msa/assets/22763846/3c053533-344f-4b92-9fde-e52cb3fd4b3d)
![image](https://github.com/nashtech-garage/nodejs-msa/assets/22763846/dbed0ae0-1f63-463b-8682-4a2d3c5bcf10)
![image](https://github.com/nashtech-garage/nodejs-msa/assets/22763846/17ee8500-128a-4429-a6c3-417bb8d1143b)
![image](https://github.com/nashtech-garage/nodejs-msa/assets/22763846/1927d5a1-fd43-4b23-9917-5e178a81594c)
![image](https://github.com/nashtech-garage/nodejs-msa/assets/22763846/5f2315e7-eda0-49b1-a325-aa99972834e2)
![image](https://github.com/nashtech-garage/nodejs-msa/assets/22763846/5cec7d83-8a42-41ca-9776-9d2f49e6467b)
